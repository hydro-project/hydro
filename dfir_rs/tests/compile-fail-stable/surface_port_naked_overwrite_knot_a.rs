../compile-fail/surface_port_naked_overwrite_knot_a.rs