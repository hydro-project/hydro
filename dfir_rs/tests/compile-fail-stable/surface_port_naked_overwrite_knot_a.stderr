error: Indexing on output is overwritten below (unknown:13:14) (1/2).
  --> tests/compile-fail-stable/surface_port_naked_overwrite_knot_a.rs:10:23
   |
10 |         out_0 = pivot[0];
   |                       ^

error: Cannot index on already-indexed output, previously indexed above (unknown:10:22) (2/2).
  --> tests/compile-fail-stable/surface_port_naked_overwrite_knot_a.rs:13:15
   |
13 |         out_0[0] -> inn_0; // Error: `pivot[0][0]`
   |               ^
